configs/milvus.yaml

```diff
@@ -45,9 +45,18 @@ etcd:
 
 metastore:
   # Default value: etcd
-  # Valid values: etcd
+  # Valid values: [etcd, tikv]
   type: etcd
 
+# Related configuration of tikv, used to store Milvus metadata.
+# Notice that when TiKV is enabled for metastore, you still need to have etcd for service discovery.
+# TiKV is a good option when the metadata size requires better horizontal scalability.
+tikv:
+  endpoints: 127.0.0.1:2379
```

Comment:
tikv port can not be 2379?

Reply:
better use default setups.
And does tikv support access cntrol?

Reply:
1. Done. Added extra comments
2. Added TLS support.